app/src/lib/components/PowerTable.svelte

```diff
@@ -840,15 +865,15 @@ onMount(async () => {
                 {#if segment_code.toLowerCase() === 'search'}
                     <div data-name="search-container" data-segment_index={segment_index}>
                         <label>
-                            <span><span>Search</span></span>
+                            <span><span>{options.translations?.search ?? 'Search'}</span></span>
                             <input data-name="search-input" type="text" placeholder=" " data-is_regex={searchObj.isRegex} data-is_custom={searchObj.isCustom} bind:value={searchObj.value} on:input={renderTable}>
                         </label>
                     </div>
                 {:else if segment_code.toLowerCase() === 'stats'}
                     <div data-name="stats-container" data-segment_index={segment_index}>
-                        {pagination.firstShownRow}-{pagination.lastShownRow} of {options.filteredRows ?? sortedData.length} 
+                        {options.translations.numeralFormatter.format(pagination.firstShownRow)}-{options.translations.numeralFormatter.format(pagination.lastShownRow)} {options.translations?.of ?? 'of'} {options.translations.numeralFormatter.format(options.filteredRows ?? sortedData.length)} 
```

Comment:
For all instances of `options.translations.numeralFormatter.format` Typescript complains that 

> 'options.translations.numeralFormatter' is possibly 'undefined'

Using optional chaining and nullish coalescing can suppress the error:

`{options?.translations?.numeralFormatter?.format(pagination.firstShownRow ?? 1)}`